org.eclipse.scout.sdk.core.s/src/main/java/org/eclipse/scout/sdk/core/s/java/apidef/Scout252Api.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2010, 2025 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.scout.sdk.core.s.java.apidef;
+
+import org.eclipse.scout.sdk.core.java.apidef.MaxApiLevel;
+
+@MaxApiLevel({25, 2})
+@SuppressWarnings({"squid:S2176", "squid:S00118", "squid:S00100", "findbugs:NM_METHOD_NAMING_CONVENTION", "squid:S2166"}) // naming conventions
+public interface Scout252Api extends IScoutApi, IScout242Api, IScoutChartApi, IScout22DoApi {
+
+  @Override
+  default int[] supportedJavaVersions() {
+    return new int[]{24};
+  }
+
+  IScoutVariousApi.JaxWsConstants JAX_WS_CONSTANTS = new Scout251Api.JaxWsConstants();
```

Comment:
I think this constant and the method "JaxWsConstants" is not necessary as it is the same as in 25.1? Can this be deleted?